sites/cheerpx/src/content/docs/12-reference/00-CheerpX-Linux-create.md

```diff
@@ -78,4 +78,116 @@ networkInterface?: NetworkInterface;
 
 This option configures network settings, which allows CheerpX to communicate over networks. For more detailed information about how CheerpX handles networking, including the use of Tailscale and overcoming browser limitations, see the [Networking](/docs/guides/Networking) guide.
 
+### `authKey`
+
+```ts
+authKey?: string;
+```
+
+The `authKey` is an optional string used for authentication and should be passed directly in the `create` call.
```

Comment:
Reword this to be more similar to the brief explanation in the networking guide above, it's more accurate

Reply:
Please remove the "passed directly into" part. It's both inaccurate (since it's passed actually inside an object) and redundant since this whole page documents the `create` API